lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
Thanks for the catch.

As for the choice in terminology... I went for flow as a very short (to type) yet still very accurate description.  I had just come from refactoring ``WebAgg`` at the time where I used the css ``flow`` property as well as following firefox's redesign from physical to logical coordinates, and so decided the name apt for here.  So yes, no special reason...